src/wallet/rpcwallet.cpp

```diff
@@ -3773,23 +3778,39 @@ UniValue getaddressinfo(const JSONRPCRequest& request)
 
     CScript scriptPubKey = GetScriptForDestination(dest);
     ret.pushKV("scriptPubKey", HexStr(scriptPubKey.begin(), scriptPubKey.end()));
+
     const SigningProvider* provider = pwallet->GetSigningProvider(scriptPubKey);
 
     isminetype mine = pwallet->IsMine(dest);
     ret.pushKV("ismine", bool(mine & ISMINE_SPENDABLE));
+
     bool solvable = provider && IsSolvable(*provider, scriptPubKey);
     ret.pushKV("solvable", solvable);
+
     if (solvable) {
        ret.pushKV("desc", InferDescriptor(scriptPubKey, *provider)->ToString());
     }
+
     ret.pushKV("iswatchonly", bool(mine & ISMINE_WATCH_ONLY));
+
+    // Return DescribeWalletAddress fields.
+    // Always returned: isscript, ischange, iswitness.
```

Comment:
nit (please don't invalidate review ACKs to change this): this commenting would be better placed next to the `DescribeWalletAddress` function or `DescribeWalletAddressVisitor` class, since it's very possible that the key-values returned will change in future, and keeping comments close to the code it's describing reduce the chances of them going stale.

Reply:
Good idea -- will update #17578 with this.

Reply:
Removed.